docs/how-to-guides/faq.md

```diff
@@ -82,15 +82,15 @@ Currently, promptflow supports the following environment variables:
 
 **PF_WORKER_COUNT** 
 
-Valid for batch run only. The number of workers to use for parallel execution of the Flow.
+Valid for batch run only. The number of workers determine the degree of parallelism of the batch run execution.
```

Comment:
It's not "valid", it should be "effective"
Setting it in non batch run is also valid, it does not cause any failure

Reply:
OK